src/aiperf/zmq/streaming_dealer_client.py

```
@@ -0,0 +1,189 @@
+
+                if self._receiver_handler:
+                    self.execute_async(self._receiver_handler(message))
+                else:
+                    self.warning(
+                        f"Received {message.message_type} message but no handler registered"
+                    )
+
+            except zmq.Again:
+                self.debug("No data on dealer socket received, yielding to event loop")
+                await yield_to_event_loop()
+            except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)